### ISSUE TITLE
Updating method of Learning rate

### ISSUE THREAD

**Issue by LB15642219539** [deepinsight/insightface#2358]
Hello, I recently studied your Arafat Loss function and your code framework. Is there anything special about the latest Learning rate adjustment method in your framework? The PolynomialLRWarmup() method is now used, while the PolyScheduler() method was previously used. I hope you can give me some help! Thank you!
